support-frontend/assets/components/page/pageScaffold.tsx

```diff
@@ -1,9 +1,10 @@
 import { css, Global } from '@emotion/react';
-import { brand, FocusStyleManager } from '@guardian/source/foundations';
+// import { brand, FocusStyleManager } from '@guardian/source/foundations';
+import { brand } from '@guardian/source/foundations';
 import type { ReactNode } from 'react';
 import CsrBanner from 'components/csr/csrBanner';
 import { SkipLink } from 'components/skipLink/skipLink';
-import { TestUserBanner } from 'components/test-user-banner/testUserBanner';
+// import { TestUserBanner } from 'components/test-user-banner/testUserBanner';
```

Comment:
This pulls in Ophan which then breaks `renderToString` by referencing `document`, so I am having a think about what we do here.